### ISSUE TITLE
Web component: Masthead-With custom data - Product & solution Service & consulting - Accessibility Checker issue.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5149]
## Detailed description

Issue:
The element with "listitem" role is not contained in or owned by an element with one of the following WAI-ARIA roles: "list, group"

Expected:
If it supposed to be a list then, role=list item should have appropriate parent role. 
Or 
If it not supposed to be a list remove the role attribute.
Example:- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role

**Comment by BrunnoM7** [comment]
Hey @Praveen-sr, would you please check if the update to the Masthead items fix this issue?

**Comment by Praveen-sr** [comment]
@BrunnoM7 Re tested this on Web components, working as expected now.
